components/EntryList.js

```diff
@@ -0,0 +1,55 @@
+import Divider from "./Divider";
+import { Fragment } from "react";
+import Tab from "./Tab";
+import Tabs from "./Tabs";
+import Badge from "./Badge";
+import FavouriteButton from "./FavouriteButton";
+
+export default function EntryList({
+  entries,
+  filter,
+  onShowAllEntries,
+  onShowFavouriteEntries,
+  allEntriesCount,
+  favouriteEntriesCount,
+  result,
+  onToggleFavourite,
+}) {
+
+  return (
+    <>
+      <h2>Journeys</h2>
+      <section name="journeys-section">
+      <Tabs>
+        <Tab onClick={onShowAllEntries} isActive={filter === "all"}>
+          Recent journeys{" "}
+          <Badge isActive={filter === "all"}>{allEntriesCount}</Badge>
+        </Tab>
+        <Tab
+          onClick={onShowFavouriteEntries}
+          isActive={filter === "favourites"}
+        >
+          Favourites{" "}
+          <Badge isActive={filter === "favourites"}>
+            {favouriteEntriesCount}
+          </Badge>
+        </Tab>
+      </Tabs>
+      <div name="journey-section__entries">
+        {entries.map((entry, index) => (
+          <Fragment key={entry.id}>
+            {index > 0 ? <Divider /> : null}
+            <FavouriteButton
+              id={entry.id}
+              isFavourite={entry.isFavourite}
+              onToggleFavourite={onToggleFavourite}
+            />
+            <p>{entry.date}, {entry.start} - {entry.destination}, {entry.km}km, {entry.transport} {entry.fuel}, {result} kg CO<sub>2</sub></p>
+          </Fragment>
+        ))}
+      </div>
+      </section>
+      <button type="submit"> + </button>
```

Comment:
this button does nothing, I guess the form you open
so maybe use a Link component with href="/"
when it should look like a button yu can use a styled component for that